test/martha/martha_v3.test.js

```diff
@@ -112,26 +142,19 @@ test.serial('martha_v3 uses the default error handler for unexpected errors', as
 });
 
 // Test the "default" case because we don't know who you are.
-test.serial('martha_v3 resolves a valid DOS-style url', async (t) => {
-    getJsonFromApiStub.onCall(0).resolves(sampleDosResponse);
-    getJsonFromApiStub.onCall(1).resolves(googleSAKeyObject);
+test.serial('martha_v3 resolves a valid DOS-style url using the "dcf-fence" provider', async (t) => {
+    const bond = bondUrls('dcf-fence');
```

Comment:
A lot of the tests near the top of this file ask Martha about `dos://abc/123` which will fall through to the default case in `determineDrsType` which uses `dcf-fence`.

If we remove that default case, we will have to update several tests to use something a little more real (but can still probably be a contrived example instead of something mimicking an actual DRS object).

Reply:
Right, though the behavior of the default case will change. Right now, we're kinda emulating martha_v2, but in the future, it'll be a more generalized martha_v3 (with no bond provider, I think, because we wouldn't know which one to use).